GitHubData/Services/IRequestRateLimitService.cs

```diff
@@ -0,0 +1,24 @@
+// <copyright file="IRequestRateLimitService.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace GitHubData.Services;
+
+/// <summary>
+/// Provides the ability to get request rate limit data for
+/// non-authorized and authorized requests to the GitHub API.
+/// </summary>
+public interface IRequestRateLimitService
+{
+    /// <summary>
+    /// Gets or sets the auth token to use when making requests for GitHub data.
```

Comment:
Refactor code docs to below:

```suggestion
    /// Gets or sets the authorization token to use when making requests for GitHub data.
```